scripts/run_server.sh

```diff
@@ -67,7 +65,7 @@ function create_eventgrid_subscription {
     --source-resource-id "$TOPIC_ID" \
     --endpoint-type=webhook \
     --endpoint="https://$1.${FRP_ADDR}${WEBHOOK_PATH}" \
-    --delivery-attribute-mapping "Authorization static $WEBHOOK_SHARED_SECRET true"
+    --delivery-attribute-mapping Authorization static $WEBHOOK_SHARED_SECRET true
```

Comment:
How does this work at all? Wouldn't it need to be quoted for the secret to be interpolated/wouldn't it treat all of these as separate arguments? I trust that it works because it's in this PR, but I'm genuinely curious about why?